src/HtmlProcessor/CssToAttributeConverter.php

```diff
@@ -23,7 +23,13 @@ final class CssToAttributeConverter extends AbstractHtmlProcessor
      * only for certain values, the mapping is an object with an allowlist
      * of nodes and values.
      *
-     * @var array<string, array{attribute: string, nodes?: array<int, string>, values?: array<int, string>}>
+     * @var array<
+     *        non-empty-string, array{
+     *          attribute: non-empty-string,
+     *          nodes?: list<non-empty-string>,
+     *          values?: list<non-empty-string>
+     *        }
+     *      >
```

Comment:
I think it would be more readable with `array{` on a separate line:


```suggestion
     * @var array<
     *        non-empty-string,
     *        array{
     *          attribute: non-empty-string,
     *          nodes?: list<non-empty-string>,
     *          values?: list<non-empty-string>
     *        }
     *      >
```